### ISSUE TITLE
[Windows, Ruby 2.5.1] Bot not running

### ISSUE THREAD

**Issue by Troesler95** [meew0/discordrb#542]
Hi all,

I noticed an issue working with this library that I figured I'd report!

---------------
## Description
When attempting to run the simple ping bot from README file (with debug enabled), I receive a few debug messages but the bot does not run (exits execution):

**code**
```ruby
# super simple bot
require 'discordrb'

bot = Discordrb::Bot.new token: '<redacted>'

bot.debug = true

bot.message(with_text: 'Ping!') do |event|
    event.respond 'Pong!'
end

bot.run
```
**output**
```powershell
PS C:\Users\troes\Documents\Dev\discord-bot> bundle exec ruby main.rb
Using WSCS version: 0.3.0
libsodium not available! You can continue to use discordrb as normal but voice support won't work.
        Read https://github.com/meew0/discordrb/wiki/Installing-libsodium for more details.
[DEBUG : main @ 2018-09-24 13:57:44.306] WS thread created! Now waiting for confirmation that everything worked
[DEBUG : websocket @ 2018-09-24 13:57:44.306] Connecting
PS C:\Users\troes\Documents\Dev\discord-bot>
```
It seems to be exiting with no errors.

After downgrading to Ruby version 2.4 (also with DevKit), the bot runs as expected.
## Environment
OS: Windows 10 1803 (build 17134)
Ruby:  ruby 2.5.1p57 (2018-03-29 revision 63029) [x64-mingw32] - installed using RubyInstaller with DevKit
Discordrb: 3.2.1 (no errors on install)

------------
Let me know if there's any more information or testing I can provide/do to help with this.

**Comment by z64** [comment]
Hi there, thanks for reporting!

This is a known issue and fixed by #478. Specifically, our `rest-client` dependency is broken with Ruby 2.5 on Windows.

You can hot-patch this yourself by running:
```
$ gem install rest-client --pre
```
or equivalent in your Gemfile based on that PR.

**Comment by Troesler95** [comment]
Ah yep that got it working! Thanks for the quick reply 😄
